Topic 2 - Ex 2/src/BlogDatabase.java

```diff
@@ -0,0 +1,44 @@
+import java.util.ArrayList;
+
+public class BlogDatabase {
+
+	private String blogName;
+	protected ArrayList<EntryDatabase> entries;
+	
+	public BlogDatabase(String name){
+		blogName = name;
+		entries = new ArrayList<EntryDatabase>();
+	}
+	
+	public String getBlogName(){
+		return blogName;
+	}
+
+	public void setBlogName(String blogName){
+		this.blogName = blogName;
+	}
+	
+	public void addNewEntry(String title, String text){
+		EntryDatabase entry = new EntryDatabase(title, text);
+		entries.add(entry);
+	}
+	
+	public void removeEntry(String title){
+		for (EntryDatabase entry : entries){

```

Comment:
Please check that you have the formatter set up and use it before submitting code, I can see theree are missing spaces in this line